src/main/java/com/fasterxml/uuid/Generators.java

```diff
@@ -178,6 +179,48 @@ public static TimeBasedGenerator timeBasedGenerator(EthernetAddress ethernetAddr
         return new TimeBasedGenerator(ethernetAddress, timer);
     }
 
+    // // DB Locality Time+location-based generation
+
+    /**
+     * Factory method for constructing UUID generator that generates UUID using
+     * variant 1 (time+location based).
```

Comment:
This should be variant 6 right? (although may refer to variant 1)

Reply:
D'oh